### ISSUE TITLE
Bug 58071649: [React Native Gallery -> Sketch]: Tooltip is not defined for color buttons present in Sketch.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#614]
Build Details:
OS Version: Dev (OS Build 27871.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.79.0

AT: Keyboard

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Media' button present in left navigation pane and expand it.
Navigate to 'Sketch' button and activate it.
Now hover or navigate to color buttons and observe.

Actual Result:

Tooltip is not defined for color buttons present in Sketch.
When user navigate or hover on the button tooltip is not appeared. 

Expected Result:
Tooltip should be defined for color buttons present in Sketch.
When user navigate or hover on the button tooltip should be visible to the user.
 
User Impact:
Users with motor impairments who rely on keyboard will face difficulty if tooltip is not defined to the button. 
 
MAS Reference:
[MAS 1.3.3 – Sensory Characteristics](https://aka.ms/MAS1.3.3)

Trap ID:
[1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)

"Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by chiaramooney** [comment]
This issue will be a little more tricky to tackle because it will involve editing the react-native-sketch-canvas module. Luckily the change should only be a JS side change (no Paper architecture specific knowledge needed).

Note: The source code for the Paper React Native Gallery begins at the root of the repo rather than inside the New Arch folder.

**Comment by vineethkuttan** [comment]
This belongs to Community Module react-native-sketch-canvas. Which won't be the part of Fabric anytime soon.

**Comment by chiaramooney** [comment]
> This belongs to Community Module react-native-sketch-canvas. Which won't be the part of Fabric anytime soon.

Will need to be fixed for Paper otherwise Paper will lose it's C (Compliant) accessibility grading.
